File: Python Excel Examples/WorkbookApiDemo/encryptDocument.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.workbook_api.WorkbookApi(configuration)

name = "EncryptDocument.xlsx"
storage = ""
folder = "ExcelDocument"
encryption = spirecloudexcel.models.WorkbookEncryptionRequest()
encryption.password = "123"
api.encrypt_document(name, encryption, folder=folder, storage=storage)

File: JavaScript Excel Examples/WorkbooksApiDemo/encryptDocument.js
(function (){
    var appId = "your id";
    var appKey = "your key";  
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudExcel= require('../../src/index');
    var apiClient = new SpirecloudExcel.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudExcel.WorkbookApi(apiClient);

    var name = "Workbook.xlsx";
    var opts = {folder : "input"};
    var encryption = new SpirecloudExcel.WorkbookEncryptionRequest();
    encryption.password = "123";
    instance.encryptDocument(name, encryption, opts,
        function (error, data, response) {
            if (error) throw error;
        });
})();